source/vibe/core/stream.d

```diff
@@ -39,12 +39,10 @@ ulong pipe(InputStream, OutputStream)(InputStream source, OutputStream sink, ulo
 	@blocking @trusted
 	if (isOutputStream!OutputStream && isInputStream!InputStream)
 {
-	import vibe.internal.allocator : theAllocator, make, dispose;
+	import vibe.internal.allocator : theAllocator, makeArray, dispose;
 
-	static struct Buffer { ubyte[64*1024] bytes = void; }
-	auto bufferobj = theAllocator.make!Buffer();
-	scope (exit) theAllocator.dispose(bufferobj);
-	auto buffer = bufferobj.bytes;
+	auto buffer = theAllocator.makeArray!ubyte(64*1024);
```

Comment:
Not sure whether it's of any concern, but `makeArray` does default initialize everything.
However, there's an optimization PR pending at Phobos: https://github.com/dlang/phobos/pull/6024

Reply:
Uninitialized would be better, let me use raw alloc maybe.